src/app/public/modules/confirm/confirm-config.ts

```diff
@@ -7,9 +7,28 @@ import {
 } from './confirm-button-config';
 
 export interface SkyConfirmConfig {
+  /**
+   * Specifies the message to display in a bold font at the top of the dialog.
+   * @required
+   */
   message: string;
+  /**
+   * Specifies secondary text to display in plain font under the primary message.
```

Comment:
If you don't read the description for `message` right before reading this, `in plain font` doesn't really makes sense? Can we just delete this to avoid confusion?

Reply:
done